lib/src/typed_queue.dart

```diff
@@ -27,17 +27,20 @@ abstract class _TypedQueue<E, L extends List<E>> with ListMixin<E> {
 
   // Iterable interface.
 
+  @override
```

Comment:
Please don't add @override.
Please don't change quotes.

(So, in general, no to all these changes).

Reply:
Done.